DecaDuino.cpp

```diff
@@ -1012,6 +1012,133 @@ uint8_t DecaDuino::getRxPrf(void) {
 	return (uint8_t)ui32t;
 }
 
+uint8_t DecaDuino::getFpAmpl1(void) {
+	
+	
+	uint8_t u8t;
+	
+	readSpiSubAddress(DW1000_REGISTER_RX_TIME, 0x07, &u8t, 1);
+	
+	
+	
+	
+	return u8t;
+}
+
+
+
+
+uint16_t DecaDuino::getFpAmpl2(void) {
+	
+	uint32_t ui32t;
+	ui32t = readSpiUint32(DW1000_REGISTER_RX_RFQUAL);
+	ui32t = ( ui32t & DW1000_REGISTER_RX_RFQUAL_FPAMPL2_MASK ) >> 16;
+	
+	return (uint16_t) ui32t;
+}
+	
+
+	
+
+uint16_t DecaDuino::getFpAmpl3(void) {
+	uint8_t buffer[2];
+	uint16_t ui16t;
+	readSpiSubAddress(DW1000_REGISTER_RX_RFQUAL, 0x06, buffer, 2);
```

Comment:
```suggestion
	readSpiSubAddress(DW1000_REGISTER_RX_RFQUAL, 0x04, buffer, 2);
```

According to the DW1000 User Manual pages 98 and 99 `FP_AMPL3` is at offset `0x04` and not `0x06.`